libs/feature-workspaces/src/lib/projects/projects.component.html

```diff
@@ -1,20 +1,38 @@
-<div *ngIf="workspace$|async as workspace">
-  <mat-list *ngIf="workspace.projects.length > 0; else empty">
-    <ng-container *ngFor="let p of workspace.projects">
-      <mat-list-item [ngClass]="p.projectType" [disableRipple]="true">
-        <div class="project-type" mat-line>{{ p.projectType }}</div>
-        <div class="project-name" mat-line>{{ p.name }}</div>
-        <div class="project-root" mat-line *ngIf="p.root">
-          <mat-icon>folder_open</mat-icon>
-          {{ p.root }}
+<ng-container *ngIf="workspace$|async as workspace">
+  <div>
+    <mat-list *ngIf="workspace.projects.length > 0; else empty">
+      <ng-container *ngFor="let p of workspace.projects">
+        <mat-list-item [ngClass]="p.projectType" [disableRipple]="true">
+          <div class="project-type" mat-line>{{ p.projectType }}</div>
+          <div class="project-name" mat-line>{{ p.name }}</div>
+          <div class="project-root" mat-line *ngIf="p.root">
+            <mat-icon>folder_open</mat-icon>
+            {{ p.root }}
+          </div>
+          <div fxLayoutWrap fxFlex="shrink" fxLayout="row" fxLayoutGap="12px" fxLayoutAlign="end center">
+            <button *ngFor="let a of p.actions" mat-stroked-button [routerLink]="a.link">
+              {{ a.actionDescription }}
+            </button>
+          </div>
+        </mat-list-item>
+        <mat-divider></mat-divider>
+      </ng-container>
+    </mat-list>
+  </div>
+</ng-container>
+
+<ng-template #empty>
+  <ui-dialog>
+      <img src="assets/code-icon.svg" height="180px">
+      <div class="headline">Empty Project!</div>
```

Comment:
Its an empty workspace which contains no projects. Can you iterate on the wording? Make it feel gentle and like everything is going to be allright. Poor fragile novice developers need to think Angular isn't so scary